lib/internal/util/inspect.js

```diff
@@ -173,13 +173,20 @@ const meta = [
 ];
 
 function getUserOptions(ctx) {
-  const obj = { stylize: ctx.stylize };
-  for (const key of ObjectKeys(inspectDefaultOptions)) {
-    obj[key] = ctx[key];
-  }
-  if (ctx.userOptions === undefined)
-    return obj;
-  return { ...obj, ...ctx.userOptions };
+  return {
```

Comment:
Can you add a comment above the declaration of `inspectDefaultOptions` that if new options are added, this should be updated?